browser_use/agent/prompts.py

```diff
@@ -138,28 +138,51 @@ def get_user_message(self, use_vision: bool = True) -> HumanMessage:
 
 
 class PlannerPrompt(SystemPrompt):
-	def get_system_message(self, is_planner_reasoning) -> Union[SystemMessage, HumanMessage]:
-		planner_prompt_text = """You are a planning agent that helps break down tasks into smaller steps and reason about the current state.
+	def __init__(self, available_actions: str):
+		self.available_actions = available_actions
+
+	def get_system_message(
+		self, is_planner_reasoning: bool, extended_planner_system_prompt: Optional[str] = None
+	) -> Union[SystemMessage, HumanMessage]:
+		"""Get the system message for the planner.
+
+		Args:
+		    is_planner_reasoning: If True, return as HumanMessage for chain-of-thought
+		    extended_planner_system_prompt: Optional text to append to the base prompt
+
+		Returns:
+		    SystemMessage or HumanMessage depending on is_planner_reasoning
+		"""
+
+		planner_prompt_text = """
+You are a planning agent that helps break down tasks into smaller steps and reason about the current state.
 Your role is to:
 1. Analyze the current state and history
 2. Evaluate progress towards the ultimate goal
 3. Identify potential challenges or roadblocks
 4. Suggest the next high-level steps to take
 
+Available actions:
+{available_actions}
```

Comment:
Adding the full actions list to the planner prompt is a bigger change than the rest, can you PR this separately so we can test the impact?